go.mod

```diff
@@ -145,7 +145,7 @@ require (
 	github.com/pelletier/go-toml/v2 v2.0.1 // indirect
 	github.com/petermattis/goid v0.0.0-20211229010228-4d14c490ee36 // indirect
 	github.com/pingcap/tipb v0.0.0-20220718022156-3e2483c20a9e // indirect
-	github.com/pkg/errors v0.9.1 // indirect
+	github.com/pkg/errors v0.9.1
```

Comment:
Don't import it

Reply:
why

Reply:
We use pingcap/errors

Reply:
Maybe because move operator error from store to pkg/errs?